tests/casr_tests/test_rust_panic/src/main.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
+    println!("Hello, world!");
```

Comment:
seems that there is no panic in this test

Reply:
test_rust_panic has a fuzz target with panic